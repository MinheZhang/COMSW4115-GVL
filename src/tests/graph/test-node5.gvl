int main() {

  node n;
  n = create_node(0.0, 1.0, 1.0, 255, 250, 245, 0);

  set_node_x(n, 10.0);
  set_node_y(n, 11.0);
  set_node_radius(n, 3.14);
  set_node_r(n, 0);
  set_node_g(n, 127);
  set_node_b(n, 255);

  printf(get_node_x(n));
  printf(get_node_y(n));
  printf(get_node_radius(n));
  printi(get_node_r(n));
  printi(get_node_g(n));
  printi(get_node_b(n));
  
  return 0;
}