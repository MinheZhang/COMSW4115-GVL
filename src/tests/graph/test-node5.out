10
11
3.14
0
127
255
